main.js

```diff
@@ -51,8 +58,9 @@ function main(args) {
         }),
         isCrossPlatform: true,
         stateTimeouts: {
-            faulty: 5 * 1000, // 5s
-            tombstone: 5 * 1000 // 5s
+            suspect: program.suspect * 1000,
```

Comment:
You could even instead of parseInt, specify a custom function to the `.option`-call that does a parseInt * 1000 for you:
```
function secondsStringToMilliseconds(s) {
    return parseInt(s, 10) * 1000;
}
...
.option('--suspect <suspect>', 'Suspect period in seconds', secondsStringToMilliseconds, 5)
```